lib/gather/gather_pool.js

```diff
@@ -37,16 +37,16 @@ const GATHER_CONFIGS = [
 	},*/
 	{
 		type: "invitational",
-		icon: '/inviteGather.png',
-		name: "Invitational Gather",
-		description: "Join on ensl.org/teams/949",
-		// Grant invite if on list
-		membershipTest: function (user) {
-			return InvitationalGather.list.some(m => m.id === user.id);
-		},
-		serverMembershipTest: function (server) {
-			return server.name.toLowerCase().indexOf("promod") === -1;
-		}
+        icon: "/inviteGather.png",
+        name: "Invitational Gather",
+        description: "You must be on a nsl team to join",
+        // Grand invite if on any nsl team
```

Comment:
Although my english isn't all to great im pretty sure it's spelled 'grant' not 'grand'
compare https://dictionary.cambridge.org/dictionary/english/grand
vs https://dictionary.cambridge.org/dictionary/english/grant